src/transformers/trainer_utils.py

```diff
@@ -111,7 +111,7 @@ class EvalPrediction:
     Parameters:
         predictions (`np.ndarray`): Predictions of the model.
         label_ids (`np.ndarray`): Targets to be matched.
-        inputs (`np.ndarray`, *optional*)
+        inputs (`np.ndarray`, *optional*):
```

Comment:
Should fail on this no? saying no doc and prediction and labels_ids shoud return to new line

Reply:
It's hard to detect there is no actual documentation, I think that's something the reviewer can spot easily.
As for return to new lines, it's only necessary for long docstrings, the syntax for predictions and label_ids is perfectly fine and accepted by the doc builder.